diff --git a/ext/toolchain/commands1.py b/ext/toolchain/commands1.py
index f32ec483..ce4213b6 100644
--- a/ext/toolchain/commands1.py
+++ b/ext/toolchain/commands1.py
@@ -779,7 +779,7 @@ class InternalCommands:
 				frameworkRootDir = "/Library/Frameworks"
 			else:
 				# TODO: auto-detect, qt can now be installed anywhere.
-				frameworkRootDir = "/Developer/Qt5.2.1/5.2.1/clang_64/lib"
+				frameworkRootDir = "##REPLACE##/Frameworks/"
 
 			target = bundleTargetDir + "/Contents/Frameworks"
 
@@ -823,7 +823,10 @@ class InternalCommands:
 						if dir.startswith("Qt"):
 							self.try_chdir(target + "/" + dir +"/Versions")
 							self.symlink("5", "Current")
-							self.move("../Resources", "5")
+							try:
+								self.move("../Resources", "5")
+							except Exception as exc:
+								print repr(exc)
 							self.restore_chdir()
 
 							self.try_chdir(target + "/" + dir)
